doc/source/io.rst

```diff
@@ -108,84 +108,81 @@ All IOs have a read() method that returns a list of :class:`Block` objects (repr
     >>> print bl[0].segments[0]
     neo.core.Segment
 
-    
+
 Read a time slice of Segment
 ============================
 
-Some object support ``time_slice`` arguments in ``read_segment()``.
-This is usefull to read only a subset of a dataset clipped in time.
+Some objects support the ``time_slice`` argument in ``read_segment()``.
+This is useful to read only a subset of a dataset clipped in time.
 By default  ``time_slice=None`` meaning load eveything.
 
-This read everything::
+This reads everything::
 
-    seg = reader.load_segment(time_slice=None)
-    shape0 = seg.analogsignals[0].shape # total shape
-    shape1 = seg.spiketrains[0].shape  # total shape
+    seg = reader.read_segment(time_slice=None)
 
-    
-This read only the first 5 seconds::
+This reads only the first 5 seconds::
 
-    seg = reader.load_segment(time_slice=(0*pq.s, 5.*pq.s))
-    shape0 = seg.analogsignals[0].shape # more small shape
-    shape1 = seg.spiketrains[0].shape  # more small shape
```

Comment:
shape0/shape1 : was this too obvious or is it too much detail ?

Reply:
I thought it was too much detail.

But I don't feel strongly about it, so no problem if you want to change it back.

Reply:
No worry it is OK like that.